### DISCUSSION TITLE
Latest Qbittorrent does not support latest Jackett plugin

### DISCUSSION THREAD

**Discussion by sonicmixmaster** [qbittorrent/search-plugins#319] [General]
I followed instructions exactly, installed latest .net runtimes, .net desktop and .net SDK, updated python to latest version, pasted api key into the .json file and qubittorrent still says jackett is not upported. It all went to shit when latest qbittorrent v5.0.0.10 (64-bit) was released and made none of the search plugins work. Even when i downgraded to prior Qbittorrent versions Jackett still dont work.

**Comment by general-research-000** [comment]
I just tried:
- qBittorrent 5.1.0
- Jacket 0.24.1465

They work together beautifully, as it appears.

**Comment by sonicmixmaster** [comment]
Why are you replying to this if you cannot use the versions I did? Posting
about a new version working does not solve the problem especially that you
are replying to a post from November 2024 when Qbittorrent did not work wit
Jackett. Qbittorrent was fixed since then but no one cared to reply when
the issue existed.

On Fri, Mar 27, 2026 at 12:19 AM general-research-000  <
